### DISCUSSION TITLE
What is secure-stun ?

### DISCUSSION THREAD

**Discussion by imtadashi** [coturn/coturn#1274] [Q&A]
How to use `secure-stun`? If I enable it, I am not able to access my Stun Server. Is there a way to pass username:password and access it?

**Comment by ggarber** [comment]
It is very uncommon to use that setting because usually it is more expensive for a server to verify the authentication than to just reply to those messages.   That's why secure-stun is not enabled in most of the deployments.

What client are you using?  uclient?

**Comment by imtadashi** [comment]
@ggarber
Sorry, I am new to stun. What do you mean by uclient? For now I have installed the STUN in my local ubuntu machine and testig them from [https://webrtc.github.io/samples/src/content/peerconnection/trickle-ice/](https://webrtc.github.io/samples/src/content/peerconnection/trickle-ice/)

**Reply by eakraly** [reply]
uclient is `turnutils_uclient` - cli tool that comes with coturn to test turnserver @nodminger
